packages/yoga/package.json

```diff
@@ -2,13 +2,15 @@
   "name": "@gympass/yoga",
   "version": "7.85.1",
   "description": "Gympass component library",
-  "main": "./src",
+  "main": "src/index.ts",
```

Comment:
Could it impact something on native? Similar to the other error we had with token PR?

Reply:
good point, I just removed `src/index.ts` and kept `./src`